class/teacher.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_service
+    true
+  end
```

Comment:
- [ ] [OPTIONAL] I am setting this requirement as optional because it may be something that you just missed, but remember what the previous reviewer said about this method and the `of_age?` method, they should be inside the `Person` class only. This is because, once you initialize your person, you are passing different arguments, such as age and parent permission, so your `can_use_service` method is going to evaluate each person once it is instantiated, so there is no need to override the method on the `Teacher` class. Hope this helps.
However, please make sure to remove this method from the `Teacher` class as well, because you will need this project for later.